### PR TITLE
Queries MongoDB

### PR DESCRIPTION
**Pull request by YuliaShvets**

### CODE REVIEW

File: src/main/java/ua/lviv/iot/parkingServer/controller/ParkingController.kt [hunk 1]
```diff
@@ -21,13 +22,30 @@ class ParkingController(private val parkingService: ParkingServiceInterface) {
...
-    fun deleteParking(@PathVariable parkingId: Long): Parking = parkingService.deleteEntity(parkingId)
+    fun deleteParking(@PathVariable parkingId: String) = parkingService.deleteEntity(parkingId)
+
+
+    @GetMapping("/location/{location}")
+    fun findParkingByLocation(@PathVariable location: String): List<Parking> {
+        return parkingService.findParkingByLocation(location)
+    }
+
+
```
Comment (ajax-gnatiuk-s) [inline src/main/java/ua/lviv/iot/parkingServer/controller/ParkingController.kt:37]: Why `findParkingByLocation` so special? You separated the function with two line breaks, but another functions are separated only with one line break.

1 line break is enough

**Review by ajax-gnatiuk-s — APPROVED** [review]
Do you plan to migrate another repositories to custom mongo queries?

File: src/main/java/ua/lviv/iot/parkingServer/controller/ParkingController.kt (hunk 1 above)
Comment (YuliaShvets) [reply src/main/java/ua/lviv/iot/parkingServer/controller/ParkingController.kt:37]: ✅

**Comment by YuliaShvets** [comment]
> Do you plan to migrate another repositories to custom mongo queries?

Yes, but not yet

**Pull request merged**